install/operator/deploy/crds/syndesis_v1alpha1_syndesis_crd.yaml

```diff
@@ -30,255 +30,115 @@ spec:
         spec:
           properties:
             addons:
-              additionalProperties:
-                additionalProperties:
-                  type: string
-                type: object
-              description: Optional add on features that can be enabled. +k8s:openapi-gen=false
+              description: Optional add on features that can be enabled.
+              properties:
+                camelk:
+                  properties:
+                    camelVersion:
+                      type: string
+                    camelkRuntime:
+                      type: string
+                    enabled:
+                      type: boolean
+                    image:
+                      type: string
+                  type: object
+                dv:
+                  properties:
+                    enabled:
+                      type: boolean
+                    resources:
+                      type: object
+                  type: object
+                jaeger:
+                  properties:
+                    enabled:
+                      type: boolean
+                    samplerParam:
+                      type: string
+                    samplerType:
+                      type: string
+                  type: object
+                ops:
+                  properties:
+                    enabled:
+                      type: boolean
+                  type: object
+                todo:
+                  properties:
+                    enabled:
+                      type: boolean
+                  type: object
               type: object
             components:
               description: Components is used to configure all the core components
                 of Syndesis
               properties:
-                db:
+                database:
                   properties:
                     database:
                       type: string
```

Comment:
Doesn't changing db make this attribute `components.properties.database.properties.database.type` which is a little repeatative?

Reply:
We can change `database.database` to `database.schema` to refer to the name of the database. I didn't like the `db` abbreviation but if other ppl find it better I can set it back to what it was .

Reply:
database.schema works for me - just avoids repetition. Better than 'db'.

Reply:
Database schema is not the same as the database name. The property should reflect what's being configured. Is it schema or database name?

Reply:
I will change it to `Name`. Final result will be: `Components.Database.Name`, `Components.Database.Password` and so on.

Reply:
https://github.com/syndesisio/syndesis/pull/7114/commits/a972c087ea3e6e929d38e84700ac23a5bf1829ab is taking care of naming